player/lua/osc.lua

```diff
@@ -1917,16 +1703,11 @@ local function osc_init()
     end
 
     ne.eventresponder["mbtn_left_up"] = function ()
-        local title = mp.get_property_osd("media-title")
-        if have_pl then
-            title = string.format("[%d/%d] %s", countone(pl_pos - 1),
-                                  pl_count, title)
-        end
-        show_message(title)
+        mp.command("show-text '${media-title} [${playlist-pos-1}/${playlist-count}]'")
```

Comment:
I didn't follow all discussion, why do we change the order here? It was before `[%d/%d] %s` and now `%s [%d/%d]`. Before is fine, no?

Reply:
That's a mistake.